templates/partials/_video.html

```diff
@@ -1,10 +1,10 @@
 {% if video.type == "youtube" %}
   <iframe id="ytplayer" type="text/html" width="818" height="460"
-          src="{{ video.url }}?autoplay=1&mute=1&cc_load_policy=1&modestbranding=1&rel=0" frameborder="0"></iframe>
+          src="" frameborder="0" data-src="{{ video.url }}?autoplay=0&mute=1&cc_load_policy=1&modestbranding=1&rel=0"></iframe>
```

Comment:
Now that the video is not autoplay you can you remove the mute option.

Reply:
Done

Reply:
Sorry I meant you could remove the mute param altogether. It is now muted when you play.

Reply:
ah, I did remove it but accidentally reverted when fixing the python linting